Docker/test/selenium/specs/pingback/pingback.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const assert = require( 'assert' );
+const defaultFunctions = require( '../../helpers/default-functions' );
+
+describe( 'Pingback', function () {
+
+	before( function () {
+		defaultFunctions();
+	} );
+
+	it( 'Should ping on first page request', function () {
+
+		browser.url( process.env.MW_SERVER + '/wiki/Main_Page' );
+
+		browser.pause( 5 * 1000 );
+
+		const sqlResult = browser.dockerExecute(
+			process.env.DOCKER_MYSQL_NAME,
+			'mysql --user ' + process.env.DB_USER +
```

Comment:
This will fail if `DB_USER` contains spaces/ quotes… but given this is test only I guess we can get away with this (also below).

Reply:
done!